roles/export-devstack-openrc/tasks/main.yaml

```diff
@@ -0,0 +1,18 @@
+- name: Register devstack openrc
+  shell:
+    cmd: |
+      source '{{ openrc_file }}' admin admin > /dev/null
+      openrc=$(env | grep OS_ | sed -e 's/^/"/' -e 's/=/": "/' -e 's/$/",/' | sed '$ s/,$//')
```

Comment:
Please add comment to clarify `sed` logic

Reply:
Sure